.github/workflows/main.yml

```diff
@@ -4,14 +4,15 @@ run-name: ${{ github.actor }} - ${{ github.ref_name}} -${{ github.sha }}
 
 on:
   push:
-    branches: ["main"]
+    branches: ["develop"]
   pull_request:
-    branches: ["main"]
+    branches: ["develop"]
```

Comment:
Here, need to keep the `main` branch, because it's a bit illogical to trigger the pipeline on a push to `develop`, but not when pushing to the `main`.

Reply:
When I use the main branch, the docker-ci and helm-ci actions do not run without merge. How can I run this actions without using the develop branch? I added a successful run of the job in my repository as the title to the PR.